import_export/resources.py

```diff
@@ -966,10 +966,22 @@ def iter_queryset(self, queryset):
         else:
             yield from queryset.iterator(chunk_size=self.get_chunk_size())
 
-    def export(self, queryset=None, *args, **kwargs):
+    def export(self, *args, queryset=None, **kwargs):
```

Comment:
I can't help but feel like this is a problem with the interface -- in Java we would probably overload this and have two different export methods I guess, but maybe we should have an export and an export_subset in a future release? I dunno. It just feels wonky.